tasks.py

```diff
@@ -252,20 +252,20 @@ def goat(_c, debug=False):
 
 
 @task
-def reformat(_c, debug=False):
-    """Reformat the goat"""
+def noreformat(_c, debug=False):
```

Comment:
I don't like noreformat, but Invoke didn't like hyphens. I'm open to suggestions if this baby is too ugly.

Reply:
@ChiefHolland if you wanted hyphens you just do something like:

```suggestion
def no_reformat(_c, debug=False):
```

and run `pipenv run invoke no-reformat`

Reply:
That said, I don't really understand the "no reformat" thing, aren't we just running the goat / standard linters?  You can update the arguments as well, remember this is just python.  So you could set a `reformat=True` argument or something.  [Here's an example](https://github.com/SeisoLLC/easy_infra/blob/4d216d4f0db4d649f1371eeeb80ebb04ef22331b/tasks.py#L644)

Reply:
Also from [the docs](https://www.pyinvoke.org/):

```
Following the lead of most Unix CLI applications, it offers a traditional flag-based style of command-line parsing, deriving flag names and value types from task signatures (optionally, of course!):

$ invoke clean --docs --bytecode build --docs --extra='**/*.pyo'
$ invoke clean -d -b build --docs -e '**/*.pyo'
$ invoke clean -db build -de '**/*.pyo'
````

Reply:
Yanked it